src/features/game/events/landExpansion/plant.ts

```diff
@@ -560,6 +548,33 @@ export function getPlotYieldAmount({
     amount += 0.2;
   }
 
+  // Generic Crop Boosts
+
+  if (isWearableActive({ name: "Infernal Pitchfork", game })) {
+    amount += 3;
+  }
+
+  amount += getBudYieldBoosts(buds ?? {}, crop);
+
+  if (isOvernightCrop(crop) && isCollectibleBuilt({ name: "Hoot", game })) {
+    amount = amount + 0.5;
+  }
+
+  // Greenhouse Crops
+  // Rice
+  if (crop === "Rice" && isWearableActive({ name: "Non La Hat", game })) {
```

Comment:
Consider: Probably future, but I think this would help readability and make the code simpler:

`isWearableBoostActive({ name: 'Non La Hat', game, ["Rice"] })`
The third parameter indicates the resource types affected and it calls `isWearableActive` inside and does the resource check as well.